### ISSUE TITLE
Installation fails in Python 3.13

### ISSUE THREAD

**Issue by dirkroorda** [annotation/stam-python#15]
Analogous to analiticcl:

```
me:~ > pip install --upgrade stam
Collecting stam
  Using cached stam-0.10.0.tar.gz (88 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: stam
  Building wheel for stam (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for stam (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [73 lines of output]
      Running `maturin pep517 build-wheel -i /Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13 --compatibility off`
      📦 Including license file "/private/var/folders/nb/_qgnfkns75bcn_p_msyjl81m0000gn/T/pip-install-e0zjm17t/stam_6be9e21509b74778819f4f90d2b73b99/LICENSE"
      🔗 Found pyo3 bindings
      🐍 Found CPython 3.13 at /Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13
      💻 Using `MACOSX_DEPLOYMENT_TARGET=11.0` for aarch64-apple-darwin by default
      warning: unused manifest key: target.aarch64-apple-darwin.rustflags
      warning: unused manifest key: target.x86_64-apple-darwin.rustflags
         Compiling proc-macro2 v1.0.86
         Compiling unicode-ident v1.0.13
         Compiling libc v0.2.159
         Compiling autocfg v1.4.0
         Compiling cfg-if v1.0.0
         Compiling serde v1.0.210
         Compiling target-lexicon v0.12.16
         Compiling once_cell v1.20.1
         Compiling typenum v1.17.0
         Compiling version_check v0.9.5
         Compiling crossbeam-utils v0.8.20
         Compiling memchr v2.7.4
         Compiling generic-array v0.14.7
         Compiling byteorder v1.5.0
         Compiling quote v1.0.37
         Compiling syn v2.0.79
         Compiling num-traits v0.2.19
         Compiling smallvec v1.13.2
         Compiling pyo3-build-config v0.20.3
         Compiling syn v1.0.109
         Compiling heck v0.4.1
         Compiling core-foundation-sys v0.8.7
         Compiling crossbeam-epoch v0.9.18
         Compiling getrandom v0.2.15
         Compiling rand_core v0.6.4
         Compiling itoa v1.0.11
         Compiling rayon-core v1.12.1
         Compiling rustix v0.38.37
         Compiling crossbeam-deque v0.8.5
         Compiling errno v0.3.9
         Compiling iana-time-zone v0.1.61
         Compiling aho-corasick v1.1.3
         Compiling indexmap v1.9.3
         Compiling crypto-common v0.1.6
         Compiling block-buffer v0.10.4
         Compiling lock_api v0.4.12
         Compiling regex-syntax v0.8.5
         Compiling bitflags v2.6.0
         Compiling ryu v1.0.18
         Compiling serde_json v1.0.128
         Compiling parking_lot_core v0.9.10
         Compiling pyo3-ffi v0.20.3
         Compiling equivalent v1.0.1
         Compiling minicbor v0.24.4
         Compiling hashbrown v0.15.0
      error: failed to run custom build command for `pyo3-ffi v0.20.3`
      
      Caused by:
        process didn't exit successfully: `/private/var/folders/nb/_qgnfkns75bcn_p_msyjl81m0000gn/T/pip-install-e0zjm17t/stam_6be9e21509b74778819f4f90d2b73b99/target/release/build/pyo3-ffi-6df9c51b5f0ef92c/build-script-build` (exit status: 1)
        --- stdout
        cargo:rerun-if-env-changed=PYO3_CROSS
        cargo:rerun-if-env-changed=PYO3_CROSS_LIB_DIR
        cargo:rerun-if-env-changed=PYO3_CROSS_PYTHON_VERSION
        cargo:rerun-if-env-changed=PYO3_CROSS_PYTHON_IMPLEMENTATION
        cargo:rerun-if-env-changed=PYO3_PRINT_CONFIG
        cargo:rerun-if-env-changed=PYO3_USE_ABI3_FORWARD_COMPATIBILITY
      
        --- stderr
        error: the configured Python interpreter version (3.13) is newer than PyO3's maximum supported version (3.12)
        = help: please check if an updated version of PyO3 is available. Current version: 0.20.3
        = help: set PYO3_USE_ABI3_FORWARD_COMPATIBILITY=1 to suppress this check and build anyway using the stable ABI
      warning: build failed, waiting for other jobs to finish...
      💥 maturin failed
        Caused by: Failed to build a native library through cargo
        Caused by: Cargo build finished with "exit status: 101": `env -u CARGO MACOSX_DEPLOYMENT_TARGET="11.0" PYO3_ENVIRONMENT_SIGNATURE="cpython-3.13-64bit" PYO3_PYTHON="/Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13" PYTHON_SYS_EXECUTABLE="/Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13" "cargo" "rustc" "--message-format" "json-render-diagnostics" "--manifest-path" "/private/var/folders/nb/_qgnfkns75bcn_p_msyjl81m0000gn/T/pip-install-e0zjm17t/stam_6be9e21509b74778819f4f90d2b73b99/Cargo.toml" "--release" "--lib" "--" "-C" "link-arg=-undefined" "-C" "link-arg=dynamic_lookup" "-C" "link-args=-Wl,-install_name,@rpath/stam.cpython-313-darwin.so"`
      Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/Library/Frameworks/Python.framework/Versions/3.13/bin/python3.13', '--compatibility', 'off'] returned non-zero exit status 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for stam
Failed to build stam
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (stam)
```

**Comment by proycon** [comment]
Yes, this is also tracked in #13, I implemented 90% today but still have a hurdle to cross.

**Comment by proycon** [comment]
Fixed in v0.10.1
